apps/docs/src/guide/creating-a-fuel-dapp/index.md

```diff
@@ -199,6 +199,8 @@ Whenever you want to add a new feature to your dApp and quickly prototype things
 
 - If you want to deploy your dApp to the testnet, check out our [Deploying a dApp to Testnet](./deploying-a-dapp-to-testnet.md) guide.
 
+- If you want to further validate the functionality of your dApp and program types, check out the `test` directory in your `create fuels` project. Couple this with our [testing guide](https://docs.fuel.network/docs/fuels-ts/testing/) to get a better understanding of how to test your dApp.
```

Comment:
Is it worth adding some documentation in around adding a new test for the `decrement_counter`?

Reply:
I've added [this](https://github.com/FuelLabs/fuels-ts/pull/2755/commits/050836c800e27c5d0475a68219f5d3c82bca4c86). Do you think that's enough or were you thinking full snippet on it's implementation?

Reply:
I think it's worth having a full snippet.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2755/commits/1296cbe8553b1b33dc524292b6a9a7f59c874085